src/data/finset/basic.lean

```diff
@@ -253,6 +253,9 @@ show (s₁ : set α) ⊂ s₂ ↔ s₁ ⊆ s₂ ∧ ¬s₂ ⊆ s₁,
 @[simp] theorem val_lt_iff {s₁ s₂ : finset α} : s₁.1 < s₂.1 ↔ s₁ ⊂ s₂ :=
 and_congr val_le_iff $ not_congr val_le_iff
 
+lemma ssubset_iff_subset_ne {s t : finset α} : s ⊂ t ↔ s ⊆ t ∧ s ≠ t :=
```

Comment:
```suggestion
lemma ssubset_iff_subset_and_ne {s t : finset α} : s ⊂ t ↔ s ⊆ t ∧ s ≠ t :=
```
to match the `lt`/`le` lemma it's proved from.

Reply:
This name I copied directly from https://github.com/leanprover-community/mathlib/blob/413b4263de879bf4f58476aebec8b28d6cdfa2c5/src/data/set/basic.lean#L257 ...

Reply:
Ok, that's a good counterargument!